3.0-RELEASE-NOTES.md

```diff
@@ -18,4 +18,10 @@ via `global.Promise`,
 you will have to check all places where you are using non-standard promise API
 and update them to use Bluebird API instead.
 
-Please see [Related code change](https://github.com/strongloop/loopback-datasource-juggler/pull/790) here.
\ No newline at end of file
+Please see [Related code change](https://github.com/strongloop/loopback-datasource-juggler/pull/790) here.
+
+## Disallow array input for updateOrCreate function
+
+Allowing updateOrCreate to accept an array as input would only work when
+creating new model instances (not updating them), so this support has been
+removed. Please use `create` function instead.
```

Comment:
Please add a link to this pull request, so that readers of the release notes can find the discussion which led to this change. See other entries for inspiration.

> Please use `create` function instead.

Perhaps we should mention another solution, and that's to use `async.each` or `Promise.all` to repeatedly call `updateOrCreate` for each instance to update.

Thoughts?